resources/assets/js/components/ApplicationReview/localizations/en.json

```diff
@@ -0,0 +1,52 @@
+    "temporary_priority_alert": "Temporary Priority Alert",
+    "get_priority_list": "Contact Talent Cloud to get a list of priorities that applied to this position. They must be reviewed and considered first.",
+    "underConsideration.description": "Review the applicants in the Veterans and Canadian Citizens section. If none or very few of these applicants meet the requirements, you can still consider non- candian - citizen applications in the Optional Consideration section",
+    "underConsideration.title": "Under Consideration",
+    "optionalConsideration.title": "Optional Consideration",
+    "optionalConsideration.description": "In this group you will find the applicants who are not Canadian Citizens or do not claim to meet the essential criteria.",
+    "screenedOut.title": "No Longer Under Consideration",
+    "screenedOut.description": "These applications have already been screened out.",
+    "screenOutAll": "Screen All Optional Candidates Out",
+    "screenOutAll.confirm": "Are you sure you want to screen out all Optional candidates?",
+    "priorityApplicants.title": "Priority Applicants",
+    "priorityApplicants.description": "These are priority applicants for this position. They must be reviewed and considered first.",
+    "veteransAndCitizens.title": "Veterans and Canadian Citizens",
+    "veteransAndCitizens.description": " ",
+    "nonCitizens.title": "Non-Canadian Citizens",
+    "nonCitizens.description": " ",
+    "essentialCriteriaFail.title": "Don't Meet Essential Criteria",
+    "essentialCriteriaFail.description": " ",
+    "toggleAccordion": "Toggle this step to view relevant applicants.",
+    "veteran": "Veteran",
+    "viewApplication": "View Application",
+    "viewApplication.title": "View this applicant's application.",
+    "viewProfile": "View Profile",
+    "viewProfile.title": "View this applicant's profile.",
+    "veteranLogo": "veteran icon",
+    "emailCandidate": "email this candidate.",
+    "decision": "Decision",
+    "notReviewed": "Not Reviewed",
+    "addNote": "+ Add a Note",
+    "editNote": "Edit Note",
+    "saving": "Saving...",
+    "saved": "Saved",
+    "save": "Save",
+    "screenedOut": "Screened Out",
+    "stillThinking": "Still Thinking",
+    "stillIn": "Still In",
+    "expandAllSkills": "Expand All Skills",
+    "collapseAllSkills": "Collapse All Skills",
+    "viewJobPoster": "View Job Poster",
+    "backToApplicantList": "< Save and Go Back to Applicant List",
+    "cancelButton": "Cancel",
+    "confirmButtion": "Confirm",
+    "screenOutConfirm": "Screen out the candidate?",
+    "screenInConfirm": "Put the candidate back in the \"under consideration\" section?",
+    "oops": "Oops...",
+    "somethingWrong": "Something went wrong while saving a review. Try again later.",
+    "copyEmailsButton": "Copy Emails",
+    "copied": "Copied!",
+    "daysSinceClosed": "EN {dayCount, plural, =0 {No Days} one {# Day} other {# Days}} Since Close"
```

Comment:
Looks like the `EN` should be removed here?

Reply:
✔️